src/debugger/titaniumDebugAdapter.ts

```diff
@@ -28,7 +27,7 @@
```

Comment:
<!--
  0 failure: 
  1 warning:  src/debugger/tita...
  
  
  DangerID: danger-id-Danger;
  File: src/debugger/titaniumDebugAdapter.ts;
  Line: 5;
-->

- :warning: src/debugger/titaniumDebugAdapter.ts line 5 – Unexpected use of 'require("url").URL'. Use the global variable 'URL' instead. (node/prefer-global/url)